### ISSUE TITLE
Accent color not used on API < 21

### ISSUE THREAD

**Issue by rubengees** [Gericop/Android-Support-Preference-V7-Fix#33]
I use Version `24.1.1.0` and everything was tested on emulators.

Consider the following screenhots:

![API 24](https://cloud.githubusercontent.com/assets/8021265/17114026/7a7fdaa2-52ad-11e6-972c-e12eab3f7016.png)

How it should be: API 24

![API 16](https://cloud.githubusercontent.com/assets/8021265/17114041/8c9d4d82-52ad-11e6-8805-e9be12af2184.png)

Not colored correctly: API 16

The preference layout is as follows:

```xml
<ListPreference
    android:defaultValue="60"
    android:dependency="pref_news_notifications"
    android:entryValues="@array/news_notifications_interval_values"
    android:key="pref_news_notifications_interval"
    android:summary="%s"
    android:title="@string/preference_news_notifications_interval_title" />
```

I don't know if this is really a bug, but I thought you may want to take a look at it.

Thanks in advance!

**Comment by Gericop** [comment]
Yeah, it's a bug introduced in v23.1.1, someone [reported it](https://code.google.com/p/android/issues/detail?id=194643) a long time ago, but Google doesn't care (again), even though it has high priority...

**Comment by Gericop** [comment]
So I checked it and via some really bad looking hacking I could change the color of the radio buttons. It basically gets the dialog view when it's being laid out and iterates over the `ListView`'s children changing the `CheckedTextView`'s compound drawables' color using color filters. I don't know if this should be included in the library as it's really-really bad looking code...

**Comment by rubengees** [comment]
Hmm, I don't know. This is a library full of hacks to fix Googles (badly written) library so that would be no problem I think. But you are in a better position to judge this and it is not a big deal, as usage of API < 21 decreases and no user is on that screen for a long time.

**Comment by Gericop** [comment]
Okay, I found something interesting about the default implementation. API 17 displays the correct colors, while API 19 doesn't, and API 21 is correct again.
**Seriously Google, just WTF?!**

**Comment by Gericop** [comment]
This bug has been resolved in the official support lib v24.2.0. Available in my library from v24.2.0.0.

**Comment by Gericop** [comment]
v24.2.0.0 has been released, update your gradle file:
```gradle
compile 'com.takisoft.fix:preference-v7:24.2.0.0'
```
